common/src/main/java/com/wynntils/models/territories/TerritoryModel.java

```diff
@@ -251,5 +277,7 @@ private void updateTerritoryProfileMap() {
                             .collect(Collectors.toSet());
                 },
                 onError -> WynntilsMod.warn("Failed to update territory data."));
+
+        scheduleTerritoryUpdate(Models.Guild.isInGuild());
```

Comment:
Shouldn't you do this in a "finally" way? So basically after processing, and after error handling as the Download object doesn't have support for "finally".